src/controllers/helpers/ethersHelper.ts

```diff
@@ -79,4 +74,13 @@ export const ethersLogToInternalEvent = <T>(
 	};
 };
 
-export { ethersEventToInternalEvent };
+export const parseTokenAttachmentEvent = (event: TokenAttachmentEvent): TokenAttachmentEventObject => {
```

Comment:
This function is contract-specific - it parses the structure of a certain contract (YlidePayV1). It is better to keep it in some utils file or a static method related to this contract. I don't think that it should be in the global `ethersHelper` file (the purpose of this file is to convert some generic ethers.js structs into the internal Ylide structs)